opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -164,26 +230,28 @@ def test_aggregated_resources_with_static_resource(self):
                 {
                     "static_key": "try_to_overwrite_existing_value",
                     "key": "value",
-                }
+                },
+                "",
             ),
         )
 
     def test_aggregated_resources_multiple_detectors(self):
         resource_detector1 = mock.Mock(spec=resources.ResourceDetector)
         resource_detector1.detect.return_value = resources.Resource(
-            {"key1": "value1"}
+            {"key1": "value1"}, ""
         )
         resource_detector2 = mock.Mock(spec=resources.ResourceDetector)
         resource_detector2.detect.return_value = resources.Resource(
-            {"key2": "value2", "key3": "value3"}
+            {"key2": "value2", "key3": "value3"}, ""
         )
         resource_detector3 = mock.Mock(spec=resources.ResourceDetector)
         resource_detector3.detect.return_value = resources.Resource(
             {
                 "key2": "try_to_overwrite_existing_value",
                 "key3": "try_to_overwrite_existing_value",
                 "key4": "value4",
-            }
+            },
+            "",
```

Comment:
Up to you, but would you try testing adding a different schema_url here or maybe in `resource_detector2`?

Reply:
Done! Put it in a separate function.